addressing.mss

```diff
@@ -1,52 +1,51 @@
+/* Features related to (postal) adresses: */
+
 @address-color: #666;
 
 #interpolation {
   [zoom >= 17] {
-    line-color: #888;
+    line-color: @address-color;
     line-width: 1;
     line-dasharray: 2,4;
   }
 }
 
-#housenumbers {
+#addresses {
   [zoom >= 17] {
-    text-name: "[addr:housenumber]";
-    text-placement: interior;
-    text-min-distance: 1;
-    text-wrap-width: 0;
-    text-face-name: @book-fonts;
-    text-fill: @address-color;
-    text-size: 10;
-    [zoom >= 20] {
-        text-size: 11;
+    text-name: "[addr_housename]";
+    ["addr_housenumber" != null] {
+      text-name: [addr_housenumber];
+      ["addr_housename" != null] {
+        text-name: [addr_housenumber] + "\n" + [addr_housename];
```

Comment:
No, as this line matches only if addr_housenumber is not null. So addr_housenumber and addr_housename must be different from null.